site/en/guide/data.ipynb

```diff
@@ -739,7 +739,7 @@
       "source": [
         "### Consuming TFRecord data\n",
         "\n",
-        "See [Loading TFRecords](../tutorials/load_data/tf_records.ipynb) for an end-to-end example.\n",
+        "See [Loading TFRecords](https://www.tensorflow.org/tutorials/load_data/tfrecord) for an end-to-end example.\n",
```

Comment:
```suggestion
        "See [Loading TFRecords](../tutorials/load_data/tfrecord.ipynb) for an end-to-end example.\n",
```

This way the link works in github and colab, as well

Reply:
`../tutorials/load_data/tfrecord.ipynb` is the original link in the colab which is throwing 404 error.